app/models/ability.rb

```diff
@@ -34,6 +34,11 @@ def initialize(account)
     # See the wiki for details:
     # https://github.com/CanCanCommunity/cancancan/wiki/Defining-Abilities
 
-    can :manage, :all if account&.role&.admin?
+    case account.role
+      when 'admin'
+        can :manage, :all
+      when 'compliance'
+        can :manage,  [Profile]
+    end
```

Comment:
It's always a good idea to catch `no role`, `new role` with `else` close in `case` statement, imo

Reply:
@mitjok do you mean I need add 'else' condition?